src/bin/server/accounts/account.rs

```diff
@@ -28,14 +28,18 @@ impl Account {
     }
 
     pub fn debit(&self, sequence: sieve::Sequence, amount: u64) -> Result<Self, Error> {
-        ensure!(self.last_sequence + 1 != sequence, InconsecutiveSequence);
+        ensure!(self.last_sequence + 1 == sequence, InconsecutiveSequence);
 
         Ok(Self {
             last_sequence: sequence,
             balance: self.balance.checked_sub(amount).context(Underflow)?,
         })
     }
 
+    pub fn last_sequence(&self) -> sieve::Sequence {
```

Comment:
Is there a place where all `pub` functions are documented? Else I'd add a small rustdoc for every `pub` function here.

Reply:
Adding it